partials/specials/banners/support-video.php

```diff
@@ -1,43 +1,73 @@
         <div class="u-video-embed-container background-red">
-          <iframe class="youtube-player lazyload" data-src="<?php echo generate_youtube_embed_url($youtube_id); ?>" frameborder="0" allowfullscreen></iframe>
+          <iframe class="youtube-player lazyload" data-src="<?php
+                                                            if ($customYoutube_id) {
+                                                              echo generate_youtube_embed_url($customYoutube_id);
+                                                            } else if ($youtube_id) {
+                                                              echo generate_youtube_embed_url($youtube_id);
+                                                            } else {
+                                                              echo generate_youtube_embed_url("c6hfjBmzt5c");
+                                                            }
+                                                            ?>" frameborder="0" allowfullscreen></iframe>
         </div>
       </div>
-      <div class="grid-item offset-s-0 is-s-24 offset-m-2 is-m-20 is-xl-12 flex-item-xxl-4">
+
+      <!-- Title and form section -->
+      <div class="support-form__inputs grid-item is-l-24 offset-xl-0 is-xxl-12">
+
+        <!-- Copy and heading -->
         <a href="<?php echo site_url('support/'); ?>">
-          <?php
-            foreach($copy as $key => $paragraph) {
-              if ($key === 0) {
-            ?>
-            <h3 class="font-size-13 font-weight-bold mb-4 js-fix-widows"><?php echo $paragraph; ?></h3>
-            <?php
+          <div class="margin-top-micro margin-bottom-small">
+            <h3 class="font-size-13 font-weight-bold mb-4 mt-4 js-fix-widows">
+              <?php
+              if ($customHeadline) {
+                echo $customHeadline;
+              } else if ($heading_copy) {
+                echo $heading_copy;
+              } else {
+                echo "Support Us";
+              }
+              ?>
+            </h3>
+          </div>
+
+          <!-- Support section text or custom message -->
+          <div class="margin-top-micro margin-bottom-small">
+            <p class="font-size-11 mb-4">
+              <?php
+              if ($cutomText) {
+                echo $cutomText;
+              } else if ($support_section_text) {
+                echo $support_section_text;
               } else {
-            ?>
-            <p class="font-size-11 mb-4"><?php echo $paragraph; ?></p>
-            <?php
+                echo "Fund truthful, independent journalism. Donate one hour’s wage per month or whatever you can afford today.";
               }
-          ?>
-          <?php
-            }
-          ?>
+              ?>
+            </p>
+          </div>
         </a>
-        <a href="<?php echo site_url('support/'); ?>" class="ui-button ui-button--white">Fund something better</a>
+        <!-- Reneders the support form -->
```

Comment:
there is some layout issue around this code. The grid isn't correct in the final output. more detail with screenshots

Reply:
I believe I have fixed the layout issues but have noticed that my fonts are different and need to look at the styles files more closely. I have added it in a to do list for us to go through to check I have installed properly.